zulip/zulip/__init__.py

```diff
@@ -1076,6 +1076,21 @@ def delete_stream(self, stream_id):
             method='DELETE',
         )
 
+    def get_member(self, user_id, request=None):
```

Comment:
Oh, let's call this `get_user`, not `get_member`.  That's more consistent naming with the rest of the product.

(And I think we'll want to rename `get_members` to `get_users`)